cross/toolchain.cmake

```diff
@@ -87,6 +87,7 @@ add_compile_param(CMAKE_MODULE_LINKER_FLAGS "${CROSS_LINK_FLAGS}" "TOOLCHAIN_EXE
 # Specify compile options
 add_compile_options("--sysroot=${CROSS_ROOTFS}")
 if (CLR_CMAKE_COMPILER STREQUAL "Clang")
```

Comment:
When I was testing this, I had it similar to your way, but didn't have the '' around the compiler name in the bash script, so I had `-DCLR_CMAKE_COMPILER=Clang`.

Reply:
ok, let me go to your version. I don't know cmake good enough to judge what is a string and what is not.